src/main/java/Board.java

```diff
@@ -3,77 +3,81 @@
 
 public class Board {
     private final Grid grid;
-
+    
```

Comment:
Did you intend to replace all empty lines with tabs?

Reply:
Thanks for spotting this, I did not. I'll change this